feat(statsd) update StatsD document (#4265)
Kong/docs.konghq.com: app/_hub/kong-inc/statsd/_index.md

## Metrics

{% if_plugin_version gte:3.0.x %}
Metric                     | Description | Namespace
---                        | ---         | ---
`request_count`            | The number of requests. | `kong.service.<service_identifier>.request.count`
`request_size`             | The request's body size in bytes. | `kong.service.<service_identifier>.request.size`
`response_size`            | The response's body size in bytes. | `kong.service.<service_identifier>.response.size`
`latency`                  | The time interval in milliseconds between the request and response. | `kong.service.<service_identifier>.latency`
`status_count`             | Tracks each status code returned in a response. | `kong.service.<service_identifier>.status.<status>`

> `unique_users`             | Tracks unique users who made requests to the underlying Service or Route. | `kong.service.<service_identifier>.user.uniques`

Reviewer: ```suggestion
`unique_users`             | Tracks unique users who made requests to the underlying Service or route. | `kong.service.<service_identifier>.user.uniques`
```

> `request_per_user`         | Tracks the request count per Consumer. | `kong.service.<service_identifier>.user.<consumer_identifier>.request.count`

Reviewer: ```suggestion
`request_per_user`         | Tracks the request count per consumer. | `kong.service.<service_identifier>.user.<consumer_identifier>.request.count`
```

`upstream_latency`         | Tracks the time in milliseconds it took for the final Service to process the request. | `kong.service.<service_identifier>.upstream_latency`
`kong_latency`             | Tracks the internal Kong latency in milliseconds that it took to run all the plugins. | `kong.service.<service_identifier>.kong_latency`

> `status_count_per_user`    | Tracks the status code for per Consumer per Service. | `kong.service.<service_identifier>.user.<consumer_identifier>.status.<status>`

Reviewer: ```suggestion
`status_count_per_user`    | Tracks the status code per consumer per service. | `kong.service.<service_identifier>.user.<consumer_identifier>.status.<status>`
```

`status_count_per_workspace`         | The status code per workspace. | `kong.service.<service_identifier>.workspace.<workspace_identifier>.status.<status>`

> `status_count_per_user_per_route`    | The status code per Consumer per Route. | `kong.route.<route_id>.user.<consumer_identifier>.status.<status>`

Reviewer: ```suggestion
`status_count_per_user_per_route`    | The status code per consumer per route. | `kong.route.<route_id>.user.<consumer_identifier>.status.<status>`
```

### {{site.base_gateway}} 3.0.x

* Merged features of the StatsD Advanced plugin into the StatsD plugin. The StatsD plugin now includes the following:
  * New parameters for StatsD: `hostname_in_prefix`, `udp_packet_size`, `ues_tcp`, `allow_status_codes`, `consumer_identifier_default`, `service_identifier_default`, `workspace_identifier_default`.
  * New metrics: `status_count_per_workspace`, `status_count_per_user_per_route`, `shdict_usage`
  * New metric fields: `service_identifier`, `workspace_identifier`

> * Breaking Changes

Reviewer: ```suggestion
* Breaking changes
```

>   * The metric name that related with the Service has been renamed by adding a `service.` prefix. e.g. `kong.service.<service_identifier>.request.count`

Reviewer: ```suggestion
  * The metric name that is related to the service has been renamed by adding a `service.` prefix. e.g. `kong.service.<service_identifier>.request.count`
```

  * The metric `kong.<service_identifier>.request.status.<status>.count` from Metrics `status_count` and `status_count_per_user` has been renamed to `kong.service.<service_identifier>.status.<status>.count`

>   * The metric `*.status.<status>.total` from Metrics `status_count` and `status_count_per_user` has been removed.

Reviewer: ```suggestion
  * The metric `*.status.<status>.total` from metrics `status_count` and `status_count_per_user` has been removed.
```
